modules/update-availability.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: CONCEPT
+[id="update-availability_{context}"]
+= Common questions about update availability
+
+There are several factors that affect if and when an update is made available to an {product-title} cluster. The following list provides common questions regarding the availability of an update:
+
+*What are the differences between each of the different update channels?*
+
+* A new release is initially added to the `candidate` channel and is often available as an update target for clusters on the `candidate` channel.
+
+* After successful internal and external testing, a release on the `candidate` channel is supported by being promoted to the `fast` channel and is often available to clusters on the `fast` channel.
+
+* After a short delay and more successful testing, a release on the `fast` channel is finally promoted to the `stable` channel and is often available to clusters on the `stable` channel. This delay represents the only difference between the `fast` and `stable` channels.
```

Comment:
This is true for 4.y releases vs. `*-4.y` channels.  For 4.(y-1).z vs. `stable-4.y`, and when `y` is even for both 4.(y-1).z and 4.(y-2).z channels vs. `eus-4.y`, there is [a generally 45-90 day delay][1], which is probably stretching "A short delay".  Not clear to me if we want to ignore that, or drop a quick hint and a reference to the more-detailed doc section, or try and clearly express that  distinction inline here, or drop "short" here, or what.

[1]: https://github.com/openshift/openshift-docs/blame/a7cd21e6501f42b882e0bd4f44131469ad66999f/modules/understanding-upgrade-channels.adoc#L95

Reply:
Agree with @wking on this . Please mention the 45-90 day delay. Point to the previous mention of this would be great.